README.md

```diff
@@ -46,6 +46,7 @@ A dbt profile can be configured to run against Trino using the following configu
 | password           | Password for authentication                                                                                  | Optional (required if `method` is `ldap` or `kerberos`)                            | `none` or `abc123`               |
 | http_headers       | HTTP Headers to send alongside requests to Trino, specified as a yaml dictionary of (header, value) pairs.   | Optional                                                                           | `X-Trino-Client-Info: dbt-trino` |
 | http_scheme        | The HTTP scheme to use for requests to Trino                                                                 | Optional (default is `http`, or `https` for `method: kerberos` and `method: ldap`) | `https` or `http`                |
+| cert               | The full path to a certificate file for authentication with trino          | Optional  |                 |
```

Comment:
nit: could you please align the columns in this line accordingly to the other rows?